library/db/src/test/kotlin/org/keynote/godtools/android/db/repository/GodToolsDaoRule.kt

```diff
@@ -0,0 +1,22 @@
+package org.keynote.godtools.android.db.repository
+
+import androidx.test.core.app.ApplicationProvider
+import io.mockk.mockk
+import org.junit.rules.TestWatcher
+import org.junit.runner.Description
+import org.keynote.godtools.android.db.GodToolsDao
+import org.keynote.godtools.android.db.GodToolsDatabase
+
+internal class GodToolsDaoRule : TestWatcher() {
```

Comment:
this was added as an intermediate step while converting/migrating. It's no longer in use, but I'm going to leave this here to aid in other tests/migrations